ui/packages/app/src/api.ts

```diff
@@ -142,6 +142,11 @@ export const fn =
 					case "send_data":
 						send_message(fn_index, payload);
 						break;
+					case "queue_full":
+						alert("Queue full. Try again later.");
```

Comment:
Instead of an alert, do you think when the feature gets matured, this could be sent as an error to the output field requested? With this same message instead of the generic "ERROR"

Reply:
Actually, could we let this message be set up by the user? If not I would suggest I would change it to something like: 
"This application is too busy! Try again soon."

Reply:
Done